contrib/docker/ckan-entrypoint.sh

```diff
@@ -58,29 +61,30 @@ link_redis_url () {
 
 # If we don't already have a config file, bootstrap
 if [ ! -e "$CONFIG" ]; then
+  write_config
+fi
```

Comment:
Why? All this does is create a fresh config if none exists. The stuff in set_environment is setting environment variables that are independent of the config file.